monitoring/monitoring/__init__.py

```diff
@@ -0,0 +1,27 @@
+from core import log
+from core.constants import (
+    TABLE_NAME_SCRAPING_AMAZON_DE,
```

Comment:
Please add a comment that and why this code is duplicated.

Reply:
Added [a697200](https://github.com/calgo-lab/green-db/pull/91/commits/a697200120e859622824a448ede0107bf381e760)